src/07-even-more-loops.js

```diff
@@ -143,7 +211,16 @@ function findEqual(array, selected) {}
  *  //> [ 1, 11, 2, 3, 4, 9 ]
  */
 
-function removeDuplicates(array) {}
+function removeDuplicates(array) {
+  let newArray = [];
+  array.forEach((duplicate) => {
```

Comment:
Nice work with this ` .forEach() ` call.  However we need to make sure that we understand exactly what this code is doing.  We have not covered higher order functions or callback functions yet.  If you can not explain what exactly this is doing all along the way lets try to solve this with only using loops.